app/services/builders/review_turnaround.rb

```diff
@@ -11,7 +13,17 @@ def call
     private
 
     def calculate_turnaround
-      @review_request.reviews.first.opened_at.to_i - @review_request.created_at.to_i
+      (opened_review_request.to_i - @review_request.created_at.to_i) - weekend_days_as_seconds
+    end
+
+    def weekend_days_as_seconds
```

Comment:
We also want this for the Merge Time calculations, so perhaps this logic should go somewhere else, where it can be reused.